v2/pom.xml

```diff
@@ -691,6 +690,9 @@
         <module>streaming-data-generator</module>
         <module>kafka-to-pubsub</module>
         <module>kafka-common</module>
+        <module>mongodb-to-bigquery</module>
```

Comment:
The modules need to be restructured a little. We now try to make all templates follow the "X-to-X" or "X-common" naming convention, where X is a "provider" like Google Cloud or MongoDB. E.g. "googlecloud-to-googlecloud", or "mongodb-to-googlecloud".

Please rename "bigquery-to-mongodb" to "googlecloud-to-mongodb", and move MongoDbToBigQuery and MongoDbToBigQueryCdc templates under "mongodb-to-googlecloud" module. If there are some common files (e.g. Options can probably be reused) they can go into a new module "mongodb-common".

The reason is that we try to avoid having too many modules, the repo is expected to grow to hundreds of templates and having a separate module for each of them is not sustainable. And in the future if you add a new templates, e.g. GCS->MongoDB it will go into an already existing module googlecloud-to-mongodb.

Some of the current modules don't follow this convention (e.g. kafka-to-pubsub), but those are older templates not yet refactored.

A good example of the new structure are elasticsearch templates:
https://github.com/GoogleCloudPlatform/DataflowTemplates/tree/main/v2/elasticsearch-common
https://github.com/GoogleCloudPlatform/DataflowTemplates/tree/main/v2/googlecloud-to-elasticsearch